Sources/Sentry/include/SentrySwizzle.h

```diff
@@ -347,33 +351,82 @@ typedef NS_ENUM(NSUInteger, SentrySwizzleMode) {
-                         mode:SentrySwizzleMode                                                    \
-                          key:KEY];
-
-#define _SentrySwizzleClassMethod(                                                                 \
-    classToSwizzle, selector, SentrySWReturnType, SentrySWArguments, SentrySWReplacement)          \
-    [SentrySwizzle                                                                                 \
-        swizzleClassMethod:selector                                                                \
-                   inClass:[classToSwizzle class]                                                  \
-             newImpFactory:^id(SentrySwizzleInfo *swizzleInfo) {                                   \
-                 SentrySWReturnType (*originalImplementation_)(                                    \
-                     _SentrySWDel3Arg(__unsafe_unretained id, SEL, SentrySWArguments));            \
-                 SEL selector_ = selector;                                                         \
-                 return ^SentrySWReturnType(_SentrySWDel2Arg(                                      \
-                     __unsafe_unretained id self, SentrySWArguments)) { SentrySWReplacement };     \
-             }];
+#if TEST
+#    define _SentrySwizzleInstanceMethod(classToSwizzle, selector, SentrySWReturnType,             \
+        SentrySWArguments, SentrySWReplacement, SentrySwizzleMode, KEY)                            \
+        [SentrySwizzle swizzleInstanceMethod:selector                                              \
+                                     inClass:[classToSwizzle class]                                \
+                               newImpFactory:^id(SentrySwizzleInfo *swizzleInfo) {                 \
+                                   SentrySWReturnType (*originalImplementation_)(_SentrySWDel3Arg( \
+                                       __unsafe_unretained id, SEL, SentrySWArguments));           \
+                                   SEL selector_ = selector;                                       \
+                                   return ^SentrySWReturnType(_SentrySWDel2Arg(                    \
+                                       __unsafe_unretained id self, SentrySWArguments)) {          \
+                                       @try {                                                      \
+                                           SentrySWReplacement                                     \
+                                       } @finally {                                                \
+                                           if (!swizzleInfo.originalCalled)                        \
+                                               @throw([NSException                                 \
+                                                   exceptionWithName:@"Swizzling Error"            \
```

Comment:
`l`: I think exception names usually don't have spaces.
```suggestion
                                                   exceptionWithName:@"SwizzlingError"            \
```